### ISSUE TITLE
rustup installer --help does not say how to separate multiple components / targets

### ISSUE THREAD

**Issue by RalfJung** [rust-lang/rustup#2064]
**Describe the problem you are trying to solve**

I am trying to install multiple components with the initial toolchain using the new `-c` flag. However, the `--help` information does not tell me how to do that. I could imagine any of the following:
```
-c component1 -c component2
-c component1 component2
-c component1,component2
```

**Describe the solution you'd like**

The `--help` text should say how to pass multiple components.

**Comment by kinnison** [comment]
I agree that it's not clear.  This is generated by clap though.  Perhaps a newer clap version will improve this?

**Comment by RalfJung** [comment]
Note that I am talking about the shell script here. Is that generated by clap as well?

Also, if you have the answer to the question, that'd really help me to fix the CI that rustup 1.20 broke. ;)

**Comment by RalfJung** [comment]
`--component rust-src --component miri` seems to work.

**Comment by kinnison** [comment]
The shell script just downloads and invokes the binary.  The help in the shell script is generated from the binary and inserted by hand, but it is for `rustup-init` not for `rustup toolchain install` which is different.

**Comment by rami3l** [comment]
Closing this as resolved since the message has been adjusted with the clap migrations:

```console
> rustup install --help
[...]
  -c, --component <COMPONENT>  Comma-separated list of components to be added on installation
  -t, --target <TARGET>        Comma-separated list of targets to be added on installation
```
